src/desktop/apps/auth2/index.ts

```diff
@@ -1,13 +1,16 @@
 import express from 'express'
+import { data as sd } from 'sharify'
 import { index } from './routes'
 // const { twitterLastStepPath } = require('@artsy/passport').options
-
 const app = (module.exports = express())
 
 app.set('view engine', 'jade')
 app.set('views', `${__dirname}/templates`)
 
 app.get('/login', index)
 app.get('/signup', index)
-app.get('/reset_password', index)
+// TODO: Remove if statement after /auth2 launches
+if (sd.NEW_AUTH_MODAL) {
+  app.get('/forgot', index)
+}
```

Comment:
This conditional is probably unnecessary since we check for the entire app [here](https://github.com/artsy/force/blob/51dc4dcd15c9e3ad16c9a5baf4ed9ce21d46eee2/src/desktop/index.js#L54-L58)

Reply:
We do actually need these because the existing `/forgot` logic lives in the `/home` app, rather than `/auth`

Reply:
Ah so, this particular one should be removed but you're right that the home one should stay.